packages/web-components/src/drawer/drawer.template.ts

```diff
@@ -0,0 +1,32 @@
+import { ElementViewTemplate, html, ref } from '@microsoft/fast-element';
+import type { Drawer } from './drawer.js';
+import { DrawerModalType } from './drawer.options.js';
+
+/**
+ * The template for the Drawer component.
+ * @public
+ */
+export function drawerTemplate<T extends Drawer>(): ElementViewTemplate<T> {
+  return html<T>`
+    <dialog
+      class="dialog"
+      part="dialog"
+      role="${x => (x.modalType === DrawerModalType.alert ? 'alertdialog' : void 0)}"
+      aria-modal="${x => (x.modalType === 'non-modal' || x.type === 'inline' ? void 0 : 'true')}"
+      aria-describedby="${x => x.ariaDescribedby}"
+      aria-labelledby="${x => x.ariaLabelledby}"
+      aria-label="${x => x.ariaLabel}"
+      size="${x => x.size}"
+      position="${x => x.position}"
+      modal-type="${x => x.modalType}"
+      type="${x => x.type}"
+      @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+      @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}"
+      ${ref('dialog')}
+    >
+      <slot></slot>
```

Comment:
Thanks for making this change, should we also include as part of this PR a `<drawer-body>` component to provide the default pattern folks can use (which would have been the previous markup within here)?